lighthouse-core/audits/byte-efficiency/next-gen-image-formats.js

```diff
@@ -25,13 +25,13 @@ const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
 const IGNORE_THRESHOLD_IN_BYTES = 8192;
 
-class UsesWebPImages extends ByteEfficiencyAudit {
+class NextGenImageFormats extends ByteEfficiencyAudit {
   /**
    * @return {LH.Audit.Meta}
    */
   static get meta() {
     return {
-      id: 'uses-webp-images',
+      id: 'next-gen-image-formats',
       title: str_(UIStrings.title),
       description: str_(UIStrings.description),
```

Comment:
We should probably update the docs to mention AVIF at some point. Maybe deprecate the WebP article and write a new one?

Reply:
> We should probably update the docs to mention AVIF at some point. Maybe deprecate the WebP article and write a new one?

👍 thats #12295